translations/PrintOsmTds.js

```diff
@@ -342,23 +342,23 @@ function dumpCsvTags(schema)
           // NOTE: Not using tColumnName here since it probably got chopped.
           if (columnName == 'ZI020_GE4')
           {
-            osmTag = 'is_in:country_code';
-            osmValue = tds70.rules.ge4List[enumValue];
+            osmTag = 'addr:country';
+            osmValue = translate.convertCountryCode('urn','c2',enumValue);
```

Comment:
This looks great, I like the use of the function instead of the TDS7 table.

Reply:
The country & language tables bake the "translate" utility script bigger but it makes finding and converting country names, cods etc easier.